app/services/support_interface/unexplained_breaks_in_work_history_export.rb

```diff
@@ -0,0 +1,108 @@
+module SupportInterface
+  class UnexplainedBreaksInWorkHistoryExport
+    class UnexplainedBreak
+      def initialize(month_range:)
+        @month_range = month_range
+      end
+
+      def start_date
+        @month_range.first.prev_month
+      end
+
+      def end_date
+        @month_range.last.next_month
+      end
+
+      def length
+        ((end_date.year * 12) + end_date.month) - ((start_date.year * 12) + start_date.month)
+      end
+    end
+
+    def data_for_export
+      data_for_export = Candidate.all.map do |candidate|
+        application_form = candidate.application_forms.order(:submitted_at).last
```

Comment:
This line is to handle the Apply Again case where a candidate has 2 applications.